api/v1alpha1/clusterextension_types.go

```diff
@@ -340,7 +340,7 @@ type CRDUpgradeSafetyPreflightConfig struct {
 	//
 	//+kubebuilder:validation:Enum:="Enabled";"Disabled"
 	//+kubebuilder:default:=Enabled
-	Policy CRDUpgradeSafetyPolicy `json:"policy"`
+	Policy CRDUpgradeSafetyPolicy `json:"policy,omitempty"`
```

Comment:
This is a required field and therefore should not have the `omitempty` JSON tag:
```suggestion
	Policy CRDUpgradeSafetyPolicy `json:"policy,"`
```